guvnor-webapp/src/main/java/org/guvnor/client/perspectives/M2RepoPerspective.java

```diff
@@ -20,13 +20,13 @@
 import javax.enterprise.event.Event;
```

Comment:
@manstis Can you verify this file? If it's correct I will send PR with another M2RepoPerspectives